### PR TITLE
Refactored according to eslint

### PR DESCRIPTION
**Pull request by tegsheeb**
In this PR, the code is refactored according to ESLint. No functional change at all.

### CODE REVIEW

File: client/src/components/TitleBar.jsx [hunk 1]
```diff
@@ -157,52 +132,85 @@ const PngIcon = styled.img`
...
+  );
 };
 
 TitleBar.propTypes = {
-  data: PropTypes.object,
-  toggleFavorite: PropTypes.func,
-
+  // eslint-disable-next-line react/forbid-prop-types
+  data: PropTypes.object.isRequired,
+  toggleFavorite: PropTypes.func.isRequired,
```
Comment (eksilva) [inline client/src/components/TitleBar.jsx:213]: Haha I disabled it for these as well

File: client/src/components/ImageGallery.jsx [hunk 2]
```diff
@@ -1,7 +1,6 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import styled from 'styled-components';
-import { Grid, Col, Row } from 'react-styled-flexboxgrid';
```
Comment (eksilva) [inline client/src/components/ImageGallery.jsx:4]: For all of these, did you have to convert everything to styled components from flexboxgrid? Or was this unused originally? Just curious

**Review by eksilva — APPROVED** [review]
Looks good, I had a just curious question about one of the imports you had taken out, but other than that, nice work!!! You definitely killed this project :)

File: client/src/components/ImageGallery.jsx (hunk 2 above)
Comment (tegsheeb) [reply client/src/components/ImageGallery.jsx:4]: It was unused originally.

**Pull request merged**